job-dsls/jobs/prod/prod_kogito_properties_generator.groovy

```diff
@@ -61,8 +61,8 @@ pipelineJob("${folderPath}/kogito-properties-generator") {
 
     parameters {
         booleanParam("IS_PROD", true, "it defines if the properties file is for prod or not")
-        stringParam("BRANCH_NAME", "master", "the branch the nightly was triggered for")
-        stringParam("REPO_URL", "http://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8081/nexus/content/repositories/kogito-master-nightly", "Prod possibility is http://rcm-guest.app.eng.bos.redhat.com/rcm-guest/staging")
+        stringParam("BRANCH_NAME", "main", "the branch the nightly was triggered for")
+        stringParam("REPO_URL", "http://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8081/nexus/content/repositories/kogito-main-nightly", "Prod possibility is http://rcm-guest.app.eng.bos.redhat.com/rcm-guest/staging")
```

Comment:
Has this been renamed/created in Nexus?

Reply:
yes -it has been created: 
https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/content/groups/kogito-main-nightly/